src/test/java/api/loans/RenewalAPITests.java

```diff
@@ -769,6 +771,36 @@ public void cannotRenewWhenItemIsNotLoanable() {
       hasLoanPolicyNameParameter("Non loanable policy"))));
   }
 
+  @Test
+  public void cannotRenewWhenItemIsDeclaredLost() {
+
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource jessica = usersFixture.jessica();
+
+    final JsonObject loanJson = loansFixture.checkOutByBarcode(smallAngryPlanet, usersFixture.jessica())
+      .getJson();
+
+    declareItemLost(loanJson);
+
+    final Response response = attemptRenewal(smallAngryPlanet, jessica);
+
+    assertThat(response.getJson(), hasErrorWith(allOf(
+      hasMessage("item cannot be renewed: item is Declared lost"),
+      hasUUIDParameter("itemId", smallAngryPlanet.getId()))));
+  }
+
+  private void declareItemLost(JsonObject loanJson) {
```

Comment:
Similarly, please move this to not-between the tests

Reply:
Done